src/ui/MessageContent/index.tsx

```diff
@@ -132,6 +132,8 @@ export default function MessageContent({
   );
   const useReplyingClassName = useReplying ? 'use-quote' : '';
 
+  const isOgMessageEnabledInGroupChannel = channel?.isGroupChannel() && config.groupChannel.enableOgtag;
```

Comment:
It seems it's another GroupChannel ogtag control part. 

I checked the type of `channel` is `GroupChannel` in [here](
https://github.com/sendbird/sendbird-uikit-react/pull/603/files#diff-0ad613792affc251d9a2e9dbe1135dec1c7ae22cb10f94be24bf9be3c34bbe87R63) but plz tell me if OpenChannel can be used here too.